Library/Formula/gotags.rb

```diff
@@ -0,0 +1,30 @@
+class Gotags < Formula
+    desc "ctags-compatible tag generator for Go"
+    homepage "https://github.com/jstemmer/gotags"
+    url "https://github.com/jstemmer/gotags/archive/v1.3.0.tar.gz"
+    sha256 "414e1f96b560b089f11f814cd9000974a8ee376bb2cd9119cce60368e89ba226"
+
+    head do
+        url "https://github.com/jstemmer/gotags.git"
+    end
+
+    depends_on "go" => :build
+
+    def install
+        ENV["GOPATH"] = buildpath
+
+        system "go", "build", "-o", "gotags"
+        bin.install "gotags"
+    end
+
+    def caveats
+        <<-EOS.undent
+        Check https://github.com/jstemmer/gotags/blob/master/README.md
+        for usage and Vim/Emacs configuration
+        EOS
+    end
+
+    test do
+        system "#{bin}/gotags", "-v"
```

Comment:
I'm not sure that it worth the effort:
a) I'm not an author, so I'm not familiar with ctags format, I basically a consumer of this tool + vim. 
b) this tool has its own set of tests, written in go 
c) I took this as an example from a lot of other tools, written in go. They are all check that this binary basically works, not how it should work.